plugins/plugin-chart-echarts/src/Treemap/transformProps.ts

```diff
@@ -62,6 +62,25 @@ export function formatLabel({
   }
 }
 
+export function formatTooltip({
+  params,
+  numberFormatter,
+}: {
+  params: TreemapSeriesCallbackDataParams;
+  numberFormatter: NumberFormatter;
+}): string {
+  const { value, treePathInfo } = params;
+  const formattedValue = numberFormatter(value as number);
+
+  const treePath = (treePathInfo ?? []).map(pathInfo => pathInfo?.name || '').filter(Boolean);
```

Comment:
There is no need to filter for null values. for instance: ["a", null, "b"] => ["a", "b"]